scripts/llmbench/generate_report.py

```diff
@@ -104,9 +96,7 @@ def calculate_ratio(values,last_values):
     ratio_inductor_llama_cpp = calculate_ratio(last_latency_dict["inductor_llama_cpp"],latency_dict["inductor_llama_cpp"])
 
     ratio_eager_gtpj_default = calculate_ratio(last_latency_dict["eager_gptj_default"],latency_dict["eager_gptj_default"])
-    ratio_eager_gtpj_cpp = calculate_ratio(last_latency_dict["eager_gptj_cpp"],latency_dict["eager_gptj_cpp"])
```

Comment:
We may need keep this 2 lines in result 3 speedup table, just replace the baseline `eager_gptj_cpp` as `eager_gptj_default` is enough. similar for llama